tasks/install.yml

```diff
@@ -78,3 +78,4 @@
     - {src: "{{ role_path }}/templates/systemd/funkwhale-api.service.j2", dest: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_api_identifier }}.service"}
     - {src: "{{ role_path }}/templates/systemd/funkwhale-celery-beat.service.j2", dest: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_celery_beat_identifier }}.service"}
     - {src: "{{ role_path }}/templates/systemd/funkwhale-celery-worker.service.j2", dest: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_celery_worker_identifier }}.service"}
+    - {src: "{{ role_path }}/templates/systemd/funkwhale-watch-imports.service.j2", dest: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_watch_imports_identifier }}.service", when: funkwhale_watch_imports_enabled}
```

Comment:
Flipping this from enabled/disabled will need to clean up in a similar way like uninstallation tasks do it. That is:

- checking if the service file exists. If yes:
  - stopping the service, if currently running
  - deleting the systemd service file

Reply:
Does it work with the following addition:
```
- name: Ensure disabled funkwhale services are stopped and removed
  ansible.builtin.include_tasks: "{{ role_path }}/tasks/remove_service.yml"
  with_items:
    - {service: "{{ funkwhale_watch_imports_identifier }}.service", path: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_watch_imports_identifier }}.service", when: not funkwhale_watch_imports_enabled}
```
Hope I'm getting the syntax of negation right.